staging/src/k8s.io/apiserver/pkg/authentication/group/authenticated_group_adder.go

```diff
@@ -51,7 +51,8 @@ func (g *AuthenticatedGroupAdder) AuthenticateRequest(req *http.Request) (*authe
 		}
 	}
 
-	r.User = &user.DefaultInfo{
+	ret := *r // shallow copy
```

Comment:
Why / how is this related to the store changes?

Reply:
not sure how I triggered it, but it broke in my e2es. Upstream merged. So we can drop in next rebase.